librocksdb_sys/src/lib.rs

```diff
@@ -976,6 +980,46 @@ extern "C" {
                                                       props: *mut DBTablePropertiesCollection,
                                                       errptr: *mut *mut c_char);
 
+    // Get All Key Versions
+    pub fn crocksdb_keyversions_create() -> *mut DBKeyVersions;
+
+    pub fn crocksdb_keyversions_destroy(kvs: *mut DBKeyVersions);
+
+    pub fn crocksdb_keyversions_iterator_create(kv: *mut DBKeyVersions)
```

Comment:
Actually, we don't know the count. We only give a begin_key and an end_key to the GetAllKeyVersions function. If we wanna get the size, we need to run the function two times.